### PR TITLE
Add a pop up for more work details

### PR DESCRIPTION
**Pull request by ChrispaixK**
# Microverse_Portfolio: Pop up
Used to display more info about my works

In this PR I :
* Add a pop-up modal in HTML
* Style the pop-up modal in CSS
* Add information in the pop up using javascript

### CODE REVIEW

**Review by ShadyShawkat — APPROVED** [review]
## Hi @ChrispaixK, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
- [ ] Kindly make sure that the works section in your HTML is empty and created dynamically through your JavaScript, as per [this](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=You%20need%20to%20update%20the%20main%20page%20so%20that%20the%20projects%20section%20is%20created%20dynamically%20using%20the%20information%20stored%20in%20that%20JavaScript%20object.%20Remember%2C%20all%20of%20the%20HTML%20in%20that%20section%20is%20created%20when%20the%20page%20loads.) requirement.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ChrispaixK** [comment]
Thank you @ShadyShawkat  for reviewing me.
I removed the content in the HTML pop-up modal, can you please help me understand the changes you are asking me to make maybe with a screenshot or anything that might help me to better understand

**Comment by ShadyShawkat** [comment]
Hey @ChrispaixK,

[This requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=You%20need%20to%20update%20the%20main%20page%20so%20that%20the%20projects%20section%20is%20created%20dynamically%20using%20the%20information%20stored%20in%20that%20JavaScript%20object.%20Remember%2C%20all%20of%20the%20HTML%20in%20that%20section%20is%20created%20when%20the%20page%20loads.) says to remove the hard coded list of projects in your HTML and to display them dynamically through you JavaScript.

- This is your hard-coded projects list in your HTML
https://github.com/ChrispaixK/Microverse_Portfolio/blob/c607347c91830def29caa70fff324b8a2e42d430/index.html#L134-L260
This code should be created dynamically through your JavaScript.
- [HINT] You can use template string to create HTML fragments easily and then append them to your list in the DOM, you can read more about this [here](https://github.com/wesbos/es6-articles/blob/master/13%20-%20Creating%20HTML%20fragments%20with%20Template%20Literals.md).

File: main.js [hunk 1]
```diff
@@ -9,4 +9,271 @@ hamburger.addEventListener('click', () => {
...
+                </p>
+              </div>
+              <div class="tech-used">
+                 <ul>
+                    ${li}
+                  </ul>
+              </div>
+                <div class="button">
+                <button id="${items[i].id}"class="work-btn" type='button' onclick=test() >See Project</button>
+                </div>
```
Comment (leonardodiasb) [inline main.js]: - [x] When I click on the "See Project" button I get the error message: Uncaught ReferenceError: test is not defined
    at HTMLButtonElement.onclick. The popup is not appearing. Kindly fix that.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @ChrispaixK , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
...
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js (hunk 1 above)
Comment (ChrispaixK) [reply main.js]: Thank you @leonardodiasb  I unfortunately noticed it after my submission :(

**Review by vechicin — APPROVED** [review]
# Hi @ChrispaixK ✋, 

Your project is complete! 🎯 🧠 💯 
There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights ⚡ 

- [x] Excellent job implementing the previous reviewers' changes!

## Optional suggestions 📂 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**